src/v2/Components/FlashBanner/__tests__/FlashBanner.jest.tsx

```diff
@@ -1,22 +1,62 @@
-import { mount } from "enzyme"
+import { ReactWrapper, mount } from "enzyme"
 import "jest-styled-components"
 import React from "react"
-import { FlashBanner } from "v2/Components/FlashBanner"
+import { BaseFlashBanner } from "v2/Components/FlashBanner"
+import { renderRelayTree } from "v2/DevTools"
+import { FlashBannerTestQueryRawResponse } from "v2/__generated__/FlashBannerTestQuery.graphql"
+import { graphql } from "react-relay"
+import { Banner } from "@artsy/palette"
+
+jest.unmock("react-tracking")
+jest.unmock("react-relay")
 
 const { location: originalLocation } = window
 
-describe("FlashBanner", () => {
-  afterEach(() => {
-    window.location = originalLocation
+const mockMe: FlashBannerTestQueryRawResponse["me"] = {
+  id: "fff",
+  canRequestEmailConfirmation: false,
+}
+
+let wrapper: ReactWrapper
+
+async function getWrapper(
+  mockData: FlashBannerTestQueryRawResponse,
+  queryString: string = ""
+) {
+  delete window.location
```

Comment:
Why add this?

Reply:
Came up while researching best practices for stubbing location. I can not doing it (i rarely use the `delete` keyword).

Reply:
Good rule of thumb with `delete` is you'll know when you need it